TTS/tts/layers/delightful_tts/variance_predictor.py

```diff
@@ -0,0 +1,91 @@
+import torch
+import torch.nn as nn  # pylint: disable=consider-using-from-import
+
+from TTS.tts.layers.delightful_tts.networks import BottleneckLayer, ConvLSTMLinear, ConvTransposed
+
+
+class VariancePredictorLSTM(nn.Module):
```

Comment:
Can you make sure everything you add is being used somewhere? I feel like `VariancePredictorLSTM` is not used for instance.